hack/images/web-terminal/release.sh

```diff
@@ -20,7 +20,7 @@ cd $(dirname $0)/../../..
 source hack/lib.sh
 
 REPOSITORY=quay.io/kubermatic/web-terminal
-VERSION=0.6.0
+VERSION=0.6.1
```

Comment:
`0.6.1` already exists against v2.23, https://github.com/kubermatic/dashboard/blob/release/v2.23/hack/images/web-terminal/release.sh#L23. You'll need to create a different tag.